tools/internal_ci/macos/grpc_basictests_objc_dbg.cfg

```diff
@@ -27,5 +27,5 @@ action {
 
 env_vars {
   key: "RUN_TESTS_FLAGS"
-  value: "-f basictests macos objc dbg --internal_ci -j 1 --inner_jobs 4 --bq_result_table aggregate_results"
+  value: "-f macos objc dbg --internal_ci -j 1 --inner_jobs 4 --bq_result_table aggregate_results"
```

Comment:
The names of the config files are still `basictests` though

Reply:
Also I recommend renaming these tests `ios` rather than `objc` so that other tests (cpp in #17058, for example) on iOS could also use it, eliminating the huge overhead like downloading master  repo of Cocoapods